apps/kimi/README.md

```
@@ -0,0 +1,129 @@
+
+Check out [ccusage: The Claude Code cost scorecard that went viral](https://www.youtube.com/watch?v=Ak6qpQ5qdgk)
+
+<p align="center">
+    <a href="https://www.youtube.com/watch?v=Ak6qpQ5qdgk">
+        <img src="https://cdn.jsdelivr.net/gh/ryoppippi/ccusage@main/docs/public/ccusage_thumbnail.png" alt="ccusage: The Claude Code cost scorecard that went viral" width="600">
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://github.com/sponsors/ryoppippi">
+        <img src="https://cdn.jsdelivr.net/gh/ryoppippi/sponsors@main/sponsors.svg">
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)